scrapers/MindGeekAPI.py

```diff
@@ -462,14 +454,14 @@ def scraping_json(api_json, url=""):
     SCENE_TITLE = re.sub(r'\sXXX|\s1080p|720p|2160p|KTR|RARBG|\scom\s|\[|]|\sHD|\sSD|', '', SCENE_TITLE)
     # Remove Date
     SCENE_TITLE = re.sub(r'\s\d{2}\s\d{2}\s\d{2}|\s\d{4}\s\d{2}\s\d{2}', '', SCENE_TITLE)
-    debug("[INFO] Title: {}".format(SCENE_TITLE))
+    log.info("Title: {}".format(SCENE_TITLE))
```

Comment:
```suggestion
    log.debug("Title: {}".format(SCENE_TITLE))
```
Only useful for debug